training/cogvideox_image_to_video_lora.py

```diff
@@ -398,6 +398,8 @@ def main(args):
 
     VAE_SCALING_FACTOR = vae.config.scaling_factor
     VAE_SCALE_FACTOR_SPATIAL = 2 ** (len(vae.config.block_out_channels) - 1)
+    RoPE_BASE_HEIGHT = transformer.config.sample_height * VAE_SCALE_FACTOR_SPATIAL
+    RoPE_BASE_WIDTH = transformer.config.sample_width * VAE_SCALE_FACTOR_SPATIAL
```

Comment:
In case of DeepSpeed we won't be able to access `transformer.config`. Let's use `model_config`?

Reply:
We can access it as this point because it is still not wrapped by deepspeed